src/applications/pensions/config/chapters/04-household-information/dependentChildInformation.js

```diff
@@ -0,0 +1,177 @@
+import get from '@department-of-veterans-affairs/platform-forms-system/get';
+import merge from 'lodash/merge';
+import moment from 'moment';
+
+import {
+  radioSchema,
+  radioUI,
+  ssnSchema,
+  ssnUI,
+  yesNoSchema,
+  yesNoUI,
+} from '@department-of-veterans-affairs/platform-forms-system/web-component-patterns';
+
+import fullSchemaPensions from 'vets-json-schema/dist/21P-527EZ-schema.json';
+import createHouseholdMemberTitle from '../../../components/DisclosureTitle';
+
+import {
+  dependentSeriouslyDisabledDescription,
+  dependentWarning,
+  disabilityDocs,
+  schoolAttendanceWarning,
+} from '../../../helpers';
+
+const { dependents } = fullSchemaPensions.properties;
+
+const childRelationshipOptions = {
+  biological: "They're my biological child",
+  adopted: "They're my adopted child",
+  stepchild: "They're my stepchild",
+};
+
+function isBetween18And23(childDOB) {
+  return moment(childDOB).isBetween(
+    moment()
+      .startOf('day')
+      .subtract(23, 'years'),
+    moment()
+      .startOf('day')
+      .subtract(18, 'years'),
+  );
+}
+
+// Checks to see if they’re under 17.75 years old
+function isEligibleForDisabilitySupport(childDOB) {
+  return moment()
+    .startOf('day')
+    .subtract(17, 'years')
+    .subtract(9, 'months')
+    .isBefore(childDOB);
+}
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    dependents: {
+      items: {
+        'ui:title': createHouseholdMemberTitle('fullName', 'Information'),
+        childPlaceOfBirth: {
+          'ui:title': 'Place of birth (city and state or foreign country)',
+        },
+        childSocialSecurityNumber: merge({}, ssnUI(), {
+          'ui:required': (formData, index) =>
+            !get(['dependents', index, 'view:noSSN'], formData),
+        }),
+        'view:noSSN': {
+          'ui:title': "Doesn't have a Social Security number",
+        },
+        childRelationship: radioUI({
+          title: "What's your relationship?",
+          // uiOptions
+          labels: childRelationshipOptions,
+        }),
+        attendingCollege: merge(
+          {},
+          yesNoUI({
+            title: 'Is your child in school?',
+            // uiOptions
+            hideIf: (formData, index) =>
+              !isBetween18And23(
+                get(['dependents', index, 'childDateOfBirth'], formData),
+              ),
+          }),
+          {
+            'ui:required': (formData, index) =>
+              isBetween18And23(
+                get(['dependents', index, 'childDateOfBirth'], formData),
+              ),
+          },
+        ),
+        'view:schoolWarning': {
+          'ui:description': schoolAttendanceWarning,
+          'ui:options': {
+            expandUnder: 'attendingCollege',
+          },
+        },
+        disabled: merge(
+          {},
+          yesNoUI({
+            title: 'Is your child seriously disabled?',
+            description: dependentSeriouslyDisabledDescription,
```

Comment:
The yesNoUI web component doesn't support a description as a parameter. You may have to revert to the yesNo widget as it was being implemented before.

Reply:
really? it looks like it does/should
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/5315216/db241c9e-31b8-43fe-a7cf-988348e28f78)

Reply:
It doesn't actually work though. I can spend some time digging into why this implementation isn't working but in the mean time we can just use the previous disabled implementation in the `dependentChildInformation.js` file. I'll add a code suggestion in the file if you'd like to commit it.